java/graphscope-sdk/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 cmake_minimum_required(VERSION 3.1)
 project(GRAPHSCOPE-JNI)
+list(APPEND CMAKE_PREFIX_PATH "/opt/graphscope")
```

Comment:
Don't include such hacks in CMakelists.txt, instead, add `-DCMAKE_PREFIX_PATH=/opt/graphscope` in your makefile when invoking `cmake`.

Reply:
Done. Pls check now.